.github/workflows/main.yml

```diff
@@ -39,6 +34,7 @@ jobs:
       - run: rustup update stable && rustup default stable
       - run: rustup component add clippy
       # Only check cargo lib for now
+      # TODO: even with `-p` clippy seems to check other packages if in a workspace.
       - run: cargo clippy -p cargo --lib -- -D warnings
```

Comment:
I think `cargo clippy -p cargo --lib --no-deps -- -D warnings` will fix the issue with `clippy` running against `cargo-util` until a follow-up PR is made.